src/rgw/rgw_sal_motr.cc

```diff
@@ -1476,24 +1492,6 @@ int MotrObject::get_obj_attrs(RGWObjectCtx* rctx, optional_yield y, const DoutPr
       ldpp_dout(dpp, 0) <<__func__<< ": Failed to get key or object's entry from bucket index. rc= " << rc << dendl;
       return rc;
     }
-  bufferlist bl;
```

Comment:
Why removed? this whole part covered in fetch_obj_entry_and_key() i think, invoked by fetch_obj_entry_and_key()

Reply:
Yes correct, this whole section is covered in get_bucket_dir_ent() which is called by fetch_obj_entry_and_key() method.

In out block of get_bucket_dir_ent method, the attribute set from motr backend is put into member object attrs.